.gitignore

```diff
@@ -8,6 +8,9 @@ npm-debug.*
 *.mobileprovision
```

Comment:
this pull request becomes easier to read by hiding whitespace
<img width="1251" alt="Screen Shot 2021-02-19 at 6 52 43 PM" src="https://user-images.githubusercontent.com/1759726/108573854-af5a3780-72e3-11eb-8afc-59a0c95843eb.png">